package.json

```diff
@@ -56,6 +56,7 @@
     "unlink:all": "node bin/unlink-all.mjs"
   },
   "dependencies": {
+    "@babel/core": "^7.24.4",
```

Comment:
Do we not need to forward the peer to consumers?

Reply:
That's a choice. We can either have it in dependencies or in peerDependencies. In dependencies is more conservative and non-breaking.

Reply:
(The path forward to getting rid of all this is when it's a v2 addon it won't try to bring any babel shenanigans at all.)